components/venue/VenueDriveTab.vue

```diff
@@ -108,78 +111,46 @@
 
 <script>
 import i18n from '@/i18n/venue/index.i18n'
-
+import VenueDivision from './VenueDivision.vue'
 export default {
     i18n,
     name: 'VenueDriveTab',
+    components: { VenueDivision },
     data() {
         return {
-            icon: {
+            parkingInfo: {
+                car: '人社館地下室設有汽車停車場，車輛進入院區持證件向大門警衛換證後可駛入院區，停車費用每小時20元。',
+            },
+        }
+    },
+    computed: {
+        icon() {
+            if (this.$i18n.locale === 'en-us') {
+                return {
+                    car: require('~/static/img/venue/car.svg'),
+                    carWhite: require('~/static/img/venue/car-white.svg'),
+                    motorWhite: require('~/static/img/venue/motor-white.svg'),
+                    parking: require('~/static/img/venue/parking.svg'),
+                    parkingPink: require('~/static/img/venue/parking-pink.svg'),
+                    venueDestination: require('~/static/img/venue/en-us/venue-destination.svg'),
+                    venueDestinationMd: require('~/static/img/venue/en-us/venue-destination-md.svg'),
+                }
+            }
+            return {
```

Comment:
What's differenet?

Reply:
venueDestination, venueDestinationMd 
image path is different when getting in different locale

Reply:
updated~